### DISCUSSION TITLE
GitHub Actions is disabled for my account: ToS violation

### DISCUSSION THREAD

**Discussion by XInTheDark** [community/community#50074] [Actions]
### Select Topic Area

Question

### Body

In February, I started seeing this message:
![](https://github.zendesk.com/attachments/token/1UvPCkMjy385SWpnchyk6wCZP/?name=image.png)

And opened a ticket in GitHub support. Today they finally responded to my ticket, however they stated that:
<img width="1029" alt="image" src="https://user-images.githubusercontent.com/61100393/225190691-80054091-197e-4d13-b48c-3c286bcd7a87.png">

In fact, they did not inform me of any details (e.g. which repository violated the ToS), as I was not notified of any violations prior to this.
That I have violated the ToS of GitHub Actions is also very strange - I have absolutely not used the Actions service itself at all.

I have replied to the ticket, asking for more details, but there is no response yet.

Can anyone help with this? Thanks in advance.

**Comment by maahaMaya** [comment]
Send below concern to GitHub support at

[developer@githubsupport.com](mailto:developer@githubsupport.com)

Hello, Mr. or Ms.
I don’t understand what happened to my account.
Would you tell me what to do now?
Thank you very much for your support.

**Reply by malesci** [reply]
I'm in the same page, have you solved in any way?

**Reply by XInTheDark** [reply]
> I'm in the same page, have you solved in any way?

Hi, GitHub support responded to my ticket relatively quickly. Turns out that one of my repositories was falsely flagged by the system, and they have manually resolved it for me. You could try contacting GitHub support as well.
